### DISCUSSION TITLE
[PvP] Allow players to join bgs as group

### DISCUSSION THREAD

**Discussion by Ottawia** [chromiecraft/chromiecraft#7076] [Suggestions]
I believe most, if not all, other active PvP players will agree that one of the best things about playing PvP is playing it with your friends, guildmates, arena partners, etc. — people you just like to play with. The most fun I have ever had in PvP was playing battlegrounds with my friends and laughing on Discord on other servers. Also, I'm speaking as someone who plays only support classes; I just enjoy that, but there is a dark side to it: you need people to support. If anyone thinks that would make battlegrounds unfair, let's at least turn it on during Call to Arms (since battlegrounds are usually full on weekends) or allow us to join in smaller groups like 2-3 players, not a full 5-man group.

**Comment by 55Honey** [comment]
Sadly the group queue system is broken in its current state and causes vastly uneven BGs and unfortunately also crashes. We're open for anyone who is willing and able to look into fixing the group queue system. The module is open sourced and there are no restrictions to look at the source code. Until that has been looked into, it needs to remain disabled to prevent frequent crashes, including rollbacks.

Regards

**Reply by heyitsbench** [reply]
What module?

**Reply by 55Honey** [reply]
https://github.com/azerothcore/mod-cfbg

**Reply by ChaoticMe** [reply]
Fully agree with the dorf. Bringing in non-pvpers and helping them have fun in a BG by teaching them has been one of the happier experiences in WoW for me. Keep us updated, Bench, if you find anything.

**Comment by QuiexVZ** [comment]
Ok, this won't work. I already can't join BG without being buillied by indiviguals, now your going to let full groups do it?

**Reply by cachocarne** [reply]
PvP is pretty much ruined on this server but it seems there isn't any interest or intention of fixing the situation. I know the day I stop playing here completely this will be one of the main reasons. It's extremely hard and tedious to gear up with the most basic pieces and while you try you are constantly facing the same fully-geared "gladiators" who are openly wintrading their arenas on World channel. The funny thing is they also complain that there's not enough people for BGs. This weekend there is CtA on Arathi (30 players BG) and games start with 2 Vs 4 or 3 Vs 3.

**Reply by QuiexVZ** [reply]
I don't understand how they fail to comprehend the basic logic. If you pick on people and bully them they won't show up. It's that simple. You'd have sociopath devoid of empathy not to realize this. I even get direct messages from people asking me to come back to WSG after being screwed in the very way i ask people not to do and they are sad I won't.

**Comment by vvranceanu** [comment]
i've had fun with friends randomly assigned in different team as well. i don't need others to feel miserable so i can have fun
